deploy/olm-catalog/node-maintenance-operator/0.1.0/node-maintenance-operator.v0.1.0.clusterserviceversion.yaml

```diff
@@ -0,0 +1,133 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  annotations:
+    alm-examples: '[{"apiVersion":"kubevirt.io/v1alpha1","kind":"NodeMaintenance","metadata":{"name":"node02"},"spec":{"reason":"Test
+      node maintenance"}}]'
+    capabilities: Basic Install
+  name: node-maintenance-operator.v0.1.0
+  namespace: placeholder
+spec:
+  apiservicedefinitions: {}
+  customresourcedefinitions:
+    owned:
+    - kind: NodeMaintenance
+      name: nodemaintenances.kubevirt.io
+      version: v1alpha1
+  description: Placeholder description
+  displayName: Node Maintenance Operator
+  install:
+    spec:
+      clusterPermissions:
+      - rules:
+        - apiGroups:
+          - ""
+          resources:
+          - services
+          - endpoints
+          - events
+          - configmaps
+          - serviceaccounts
+          verbs:
+          - '*'
+        - apiGroups:
+          - ""
+          resources:
+          - nodes
+          verbs:
+          - get
+          - list
+          - update
+          - patch
+          - watch
+        - apiGroups:
+          - ""
+          resources:
+          - pods
+          verbs:
+          - get
+          - list
+          - watch
+        - apiGroups:
+          - ""
+          resources:
+          - pods/eviction
+          verbs:
+          - create
+        - apiGroups:
+          - ""
+          resources:
+          - namespaces
+          verbs:
+          - get
+        - apiGroups:
+          - apps
+          resources:
+          - deployments
+          - daemonsets
+          - replicasets
+          - statefulsets
+          verbs:
+          - '*'
+        - apiGroups:
+          - monitoring.coreos.com
+          resources:
+          - servicemonitors
+          verbs:
+          - get
+          - create
+        - apiGroups:
+          - kubevirt.io
+          resources:
+          - '*'
+          verbs:
+          - '*'
+        - apiGroups:
+          - oauth.openshift.io
+          resources:
+          - '*'
+          verbs:
+          - '*'
+        serviceAccountName: node-maintenance-operator
+      deployments:
+      - name: node-maintenance-operator
+        spec:
+          replicas: 1
+          selector:
+            matchLabels:
+              name: node-maintenance-operator
+          strategy: {}
+          template:
+            metadata:
+              labels:
+                name: node-maintenance-operator
+            spec:
+              containers:
+              - env:
+                - name: WATCH_NAMESPACE
+                - name: POD_NAME
+                  valueFrom:
+                    fieldRef:
+                      fieldPath: metadata.name
+                - name: OPERATOR_NAME
+                  value: node-maintenance-operator
+                image: quay.io/kubevirt/node-maintenance-operator:latest
```

Comment:
is this correct? already moved under kubevirt umbrealla? (I'm asking, I'm not up to date)

Reply:
@fromanirh  yes, i got a repo created + creds.  now we can keep track and tags

Reply:
@ksimon1 we might need align the image eventually to a specific release tag. 
maybe even worth generating the tag again a env variable.

Reply:
yes! Please do not use latest

Reply:
I use "latest" myself, will fix that ASAP

Reply:
This file is auto generated by operator-sdk. So if we want to change it, it has to be changed in deploy/operator.yaml. And there is the question, how we would like to change it.